sentry_sdk/client.py

```diff
@@ -109,6 +109,13 @@ def _get_options(*args, **kwargs):
     if rv["environment"] is None:
         rv["environment"] = os.environ.get("SENTRY_ENVIRONMENT") or "production"
 
+    if rv["debug"] is None:
+        rv["debug"] = os.environ.get("SENTRY_DEBUG", "False").lower() in (
+            "true",
+            "1",
+            "t",
+        )
```

Comment:
I kinda remember that we added a helper for determining if a config option was set to something truthy. Can't find it off the top of my head but will look a bit more 👀